Tests/BotTests/MergeServiceTests.swift

```diff
@@ -648,25 +648,25 @@ class MergeServiceTests: XCTestCase {
 
         let pullRequests = (1...3)
             .map {
-                PullRequestMetadata.stub(number: $0, labels: [.init(name: IntegrationLabel)])
+                PullRequestMetadata.stub(number: $0, labels: [integrationLabel])
                     .with(mergeState: .clean)
         }
 
         perform(
             stubs: [
                 .getPullRequests { pullRequests.map { $0.reference } },
                 .getPullRequest { _ in pullRequests[0] },
-                .postComment { message, number in
+                .postComment { message, pullRequest in
                     expect(message) == "Your pull request was accepted and is going to be handled right away 🏎"
-                    expect(number) == 1
+                    expect(pullRequest.number) == 1
                 },
-                .postComment { message, number in
+                .postComment { message, pullRequest in
                     expect(message) == "Your pull request was accepted and it's currently `#2` in the queue, hold tight ⏳"
-                    expect(number) == 2
+                    expect(pullRequest.number) == 2
                 },
-                .postComment { message, number in
+                .postComment { message, pullRequest in
                     expect(message) == "Your pull request was accepted and it's currently `#3` in the queue, hold tight ⏳"
-                    expect(number) == 3
+                    expect(pullRequest.number) == 3
```

Comment:
It's disturbing that the number/position in the queue happen to be the exact same as the PR number. At first I thought "wait, `pullRequest.number` is supposed to contain the order of the PR in the queue, not the GitHub PR number?" then I realised it was just a coincidence... but if it happens that we have a regression that makes it accidentally happen (e.g. `pullRequest.number` accidentally assigned to the queue position instead of GH PR ID), we will not catch the error... so maybe just invert the order in which you merge the PRs in the stub, or affect unrelated/random PR numbers to avoid that confusion?